### PR TITLE
FDG-9492 Adding Missing Events

### PR DESCRIPTION
**Pull request by chasls2**
https://federal-spending-transparency.atlassian.net/browse/FDG-9492

No test coverage change

### CODE REVIEW

File: static/data/ga-event-tracking/deficitExplainerEventTracking.csv [hunk 1]
```diff
@@ -11,21 +11,21 @@
...
 12,"Click on Deficit tab with the ""Understanding the National Deficit"" section",Fiscal Data - Explainers,Tab Click,Deficit - Deficit
-13,Click on link to MTS dataset,Fiscal Data - Explainers,Citation Click,Deficit
-14,"Click on link to USAspending.gov, titled ""the federal response to COVID-19""",Fiscal Data - Explainers,Citation Click,Deficit
+13,Click on link to MTS dataset,Fiscal Data - Explainers,Deficit Citation Click,Deficit
+14,"Click on link to USAspending.gov, titled ""the federal response to COVID-19""",Fiscal Data - Explainers,Deficit Citation Click,Deficit
 15,"Expand the accordion labeled ""How else does the federal government finance a deficit?""",Fiscal Data - Explainers,Accordion Expand Click,Deficit - The Difference Between the National Deficit and the National Debt
 16,"Collapse the accordion labeled ""How else does the federal government finance a deficit?""",Fiscal Data - Explainers,Accordion Collapse Click,Deficit - The Difference Between the National Deficit and the National Debt
 17,"Click on link to USAspending.gov, titled ""the federal response to COVID-19""",Fiscal Data - Explainers,Citation Click,Deficit
```
Comment (jdach) [inline static/data/ga-event-tracking/deficitExplainerEventTracking.csv:18]: This citation click should also be updated

**Comment by chasls2** [comment]
AFG overview was in another ticket. I updated master so it should so those changes. and fix that one missing deficit click citation

**Comment by jdach** [comment]
> AFG overview was in another ticket. I updated master so it should so those changes. and fix that one missing deficit click citation

Sounds good!

**Review by jdach — APPROVED** [review]
Changes look great

**Pull request merged**